views/well_uploader.py

```diff
@@ -71,23 +77,7 @@ def post(self, request, *args, **kwargs):
                 elif str(gw_location_file).split('.')[-1] == "xlsx":
```

Comment:
hi @ann26 please handle this, if the extension is not accepted, return bad request